mem_mgmt/mem_mgmt.h

```diff
@@ -46,7 +46,7 @@
 
 #define RAM_ADDR        (0x20000000u)
 #define NVM_ADDR        (0x60000000u)
-#define CITIROC_OFFSET  (0xF010u)
+#define CITIROC_OFFSET  (0x40000u)
```

Comment:
This will lead to a run-time error, as this address does not exist (at least if I check the MSS User Guide). The device present on the CUBES PCB is an M2S010 device, which does not have eNVM_1 (which would start at 0x60040000).

Suggest you use another offset.